api/src/certification/configuration/domain/usecases/get-sco-blocked-access-dates.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef {import ('./index.js').scoBlockedAccessDatesRepository} scoBlockedAccessDatesRepository
+ */
+
+/**
+ * @param {Object} params
+ * @param {scoBlockedAccessDatesRepository} params.scoBlockedAccessDatesRepository
+ * @returns {Promise<Array<>>}
```

Comment:
Si possible de préciser `Array<>` de quel type (dates, string..)


```suggestion
 * @returns {Promise<Array<ScoBlockedAccessDate>>}
```

Reply:
Corrigé